app/views/articles/index.rss.builder

```diff
@@ -0,0 +1,21 @@
+xml.instruct! :xml, version: "1.0"
+xml.rss version: "2.0", "xmlns:atom": "http://www.w3.org/2005/Atom" do
+  xml.channel do
+    xml.title "SumOfUs"
+    xml.description "stopping big corporations from behaving badly."
+    xml.link articles_url
+
+    @pages.each do |article|
+      xml.item do
+        xml.title article.title
+        xml.description do
+          desc = simple_format article.content
+          xml.cdata! desc
+        end
+        xml.pubDate article.created_at.to_s(:rfc822)
```

Comment:
I think here, we do need `updated_at` instead (just guessing from [this](https://webmasters.googleblog.com/2014/10/best-practices-for-xml-sitemaps-rssatom.html)). i.e. we show the last modification in `pubDate` even though we're sorting by `created_at` to keep the feed list meaningful.